src/quo2/components/list_items/user_list/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.list-items.user-list.view
+  (:require [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.icon :as icons]
+            [quo2.components.list-items.user-list.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [quo2.components.selectors.selectors :as selectors]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.utils.utils :as utils]))
+
+(def ^:private icon-types {:options :i/options
+                           :close   :i/close})
+
+(defn- cant-be-invited-msg []
+  [rn/view {:style               style/cant-be-invited
+            :accessibility-label "user can't be invite"}
+   [icons/icon :i/info {:size            12
+                        :color           colors/neutral-40
+                        :container-style style/icon-container}]
+   [text/text {:weight :regular
+               :size   :label
+               :style  style/cant-be-invited-text-style}
+    "Can’t be invited due to their privacy settings"]])
+
+(defn- right-section [{:keys [id button-type on-press checked?]}]
+  (if (= button-type :checkbox)
+    [selectors/checkbox {:default-checked? checked?}]
+    [rn/touchable-opacity
+     {:on-press            on-press
+      :accessibility-label (str (or id "user-list") "-" (if (= button-type :options)
```

Comment:
I believe accessibility labels should be very human readable. Adding the ID to it doesn't seem very helpful when using a screen reader.

On top of that @ibrkhalil, not something for this PR, but for future discussions, I think our current conventions for accessibility labels might not be the best, i.e. using keywords with dashes separating words and often being very short seems to be very machine oriented. I'd love to hear more about this from people who use these labels, particularly in RN apps. I'd also be interesting to see how other accessibility properties affect screen readers and how we can better use them (e.g. https://reactnative.dev/docs/accessibility#accessibilityhint).

Reply:
I am interested to know how screen readers handle dashes

Reply:
Me too, I'll try to learn more about this soon.